dspace-server-webapp/src/test/java/org/dspace/app/rest/StatisticsRestRepositoryIT.java

```diff
@@ -207,7 +216,7 @@ public void usagereport_loggedInUserReadRights() throws Exception {
         getClient(loggedInToken).perform(
             get("/api/statistics/usagereports/" + itemNotVisitedWithBitstreams.getID() + "_" + TOTAL_VISITS_REPORT_ID))
                                 // ** THEN **
-                                .andExpect(status().isOk());
+                                .andExpect(status().isForbidden());
```

Comment:
Did we move the test for `isOk()` elsewhere now?  I just want to be sure we didn't lose that test on accident.  We should make sure loggedInToken still works when the setting is `false`